### PR TITLE
updated readme to have the chocolatey link

### PR DESCRIPTION
**Pull request by cinemassacres**
Pull request checklist:

- [ ] `CHANGELOG.md` was updated, if applicable

# Description
when they accept it this should be merged

### CODE REVIEW

**Comment by alazymeme** [comment]
@fourtf do you want the commands put directly onto the repo? Or just link to the package page?

File: README.md [hunk 1]
```diff
@@ -35,7 +35,6 @@ git submodule update --init --recursive
 [Building on Mac](../master/BUILDING_ON_MAC.md)
 
 [Building on FreeBSD](../master/BUILDING_ON_FREEBSD.md)
-
```
Comment (alazymeme) [inline README.md:38]: Irrelevant change

File: README.md [hunk 2]
```diff
@@ -52,3 +51,5 @@ Qt creator should now format the documents when saving it.
 
 ## Doxygen
 Doxygen is used to generate project information daily and is available [here](https://doxygen.chatterino.com).
+## chocolatey
```
Comment (alazymeme) [inline README.md:54]:
```suggestion

## Chocolatey 
```

**Comment by TranRed** [comment]
why does this have 11 commits for 2 lines of text, tho ?

FarmingCommits

**Comment by cinemassacres** [comment]
Closes #1981

**Comment by apa420** [comment]
Fix your commit tree...

**Comment by alazymeme** [comment]
Unsure as to why my comments were marked as resolved when they evidently are not 🤔

File: README.md (hunk 2 above)
Comment (alazymeme) [inline README.md:54]:
```suggestion

## Chocolatey
```

File: README.md (hunk 1 above)
Comment (alazymeme) [inline README.md:38]: Irrelevant change, please revert this commit

**Review by alazymeme — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alazymeme) [reply README.md:54]: Please keep the formatting consistent.

File: README.md [hunk 3]
```diff
@@ -52,3 +51,5 @@ Qt creator should now format the documents when saving it.
 
 ## Doxygen
 Doxygen is used to generate project information daily and is available [here](https://doxygen.chatterino.com).
+## chocolatey 
+ A chocolatey version is available [here](https://chocolatey.org/packages/chatterino).
```
Comment (RAnders00) [inline README.md:55]: The chocolatey section should be next to the Download section, not at the bottom next to things it isn't related to.

**Review by RAnders00 — CHANGES_REQUESTED** [review]

**Comment by RAnders00** [comment]
Sorry, I don't mean to be rude, but this change so incredibly minute that it's more work for us to suggest changes to you than it is worth it. For simplicity's sake, I will close this, and we will introduce a chocolatey link in the readme via #2017.

**Pull request closed without merging**